src/otx/config/base/detection.yaml

```diff
@@ -0,0 +1,4 @@
+defaults:
+  - default
```

Comment:
I'm not sure if I like this approach. I think this is one of the main limitations of mmx ecosystem. When I look at a mmx config file, I found extremely annoying to navigate around to find the base config stuff. I understand that this is to avoid duplication; however, there should be trade-off between duplication and abstraction.

This will make the config navigation so difficult that when the reader needs to see a parameter, they will need to navigate around to see what is inside this default stuff.

In fact, even mmx developers agreed to this and introduced a new feature to make this a bit easier.